src/screens/WikiFilter/index.tsx

```diff
@@ -0,0 +1,299 @@
+            if (isChecked.groups[i].activate) {
+                if (new_url)
+                    new_url += "&"
+                new_url += "largeGroup=" + isChecked.groups[i].name;
+                for (let j = 0; j < isChecked.groups[i].subGroups.length; j++) {
+                    if (isChecked.groups[i].subGroups[j].activate)
+                        new_url += "&group=" + isChecked.groups[i].subGroups[j].name;
+                }
+            }
+        }
+        if (new_url)
+            new_url = '?' + new_url;
+
+        const resetAction = CommonActions.reset({
+            index: 0,
+            routes: [{
+                name: 'WikiFishlogs',
+                params: {
+                    filterQuery: new_url,
+                },
+            }],
+        });
+        navigation.dispatch(resetAction);
+    }
+
+    const clearFilter = () => {
+        const newState = { ...isChecked };
+        for (let i = 0; i < newState.groups.length; i++) {
+            newState.groups[i].activate = false;
+            for (let j = 0; j < newState.groups[i].subGroups.length; j++) {
+                newState.groups[i].subGroups[j].activate = false;
+            }
+        }
+        setIsChecked(newState);
+
+    }
+    
+    const subGroupList = (group: IGroup, g_index: number) => {
+        return group.groups[g_index].subGroups.map((item, index) => {
+            return (
+                <CheckBoxRow key={`subg${index}`}>
+                    <Checkbox
+                        value={item.activate}
+                        onValueChange={() => {
+                            const newState = { ...isChecked };
+                            newState.groups[g_index].subGroups[index].activate = !newState.groups[g_index].subGroups[index].activate;
+                            setIsChecked(newState);
+                        }}
+                        color={item.activate ? '#00BBD4' : undefined}
+                    />
+                    <RegularText text={item.name} />
+                </CheckBoxRow>
+            );
+        });
+    }
+
+    const groupList = () => {
+        return isChecked.groups.map((item, index) => {
+            return (
+                <CheckBoxRow key={`g${index}`}>
+                    <Checkbox
+                        value={item.activate}
+                        onValueChange={() => {
+                            const newState = { ...isChecked };
+                            newState.groups[index].activate = !newState.groups[index].activate;
+                            setIsChecked(newState);
+                        }}
+                        color={item.activate ? '#00BBD4' : undefined}
+                    />
+                    <RegularText text={item.name} />
+                </CheckBoxRow>
+            );
+        });
+    }
+
+    return (
+        <PageContainer>
+            <TopBar title='Filtros' icon="delete" iconText="Limpar" buttonFunction={clearFilter} />
```

Comment:
Adicionar o botão de voltar manualmente, ou criar uma topbar no [navigation personalizada](https://reactnavigation.org/docs/headers/#replacing-the-title-with-a-custom-component) com o botão de excluir